ambient.d/trains/AT_OEBB.fish

```diff
@@ -0,0 +1,28 @@
+ambient_is_ssid "OEBB"; or exit
+
+curl -s https://railnet.oebb.at/assets/modules/fis/combined.json | jq -r '"
+AMBIENT_AT_OEBB_SPEED=\(.operationalMessagesInfo.speed)
+AMBIENT_AT_OEBB_SITUATION=\(.operationalMessagesInfo.situation)
+AMBIENT_AT_OEBB_DELAY=\(.operationalMessagesInfo.delay)
+AMBIENT_AT_OEBB_SPEED_UNIT=km/h
+AMBIENT_AT_OEBB_LATITUDE=\(.mapInfo.latitude)
+AMBIENT_AT_OEBB_LONGITUDE=\(.mapInfo.longitude)
+AMBIENT_AT_OEBB_ORIENTATION=\(.mapInfo.orientation)
+AMBIENT_AT_OEBB_SERVER_TIME=\(.operationalMessagesInfo.time | strptime("%s") | todate)
+AMBIENT_AT_OEBB_TRAIN_TYPE=\(.currentJourney.trainType)
+AMBIENT_AT_OEBB_VZN=\(.currentJourney.tripNumber)
```

Comment:
I think this should actually be called `TRIP_NUMBER`, I don't actually know what `vzn` means, but I assume it's some sort of German abbreviation that DB like.

Reply:
There was also a field called lineNumber which seemed to have the same contents, so I've added that in as well